assets/app/view/hex.rb

```diff
@@ -89,5 +90,16 @@ def on_hex_click(event)
         @tile_selector.tile = @tile
       end
     end
+
+    def opacity(layable)
+      corp_track_laying = @round.is_a?(Engine::Round::Operating) && (@round.step == 'track')
+      company_track_laying = @round.is_a?(Engine::Round::Special)
```

Comment:
@tobymao admittedly I'm not sure if `company_track_laying` should have another check like `@round.can_lay_track?`, but this seemed to work correctly for all privates, i.e., it faded out most of the map after clicking on a private that could lay track, and didn't fade the map after clicking on the Dougo to exchange for a share of IR.